### PR TITLE
third commit.

### PR DESCRIPTION
**Pull request by Herreran903**
Pull request to modified the class HolaMundo. Please review and merge.

### CODE REVIEW

File: HolaMundo.java [hunk 1]
```diff
@@ -3,4 +3,5 @@ public class HolaMundo {
     private int a;
     private int b;
     private int c;
+    private int e;
```
Comment (kamen7375) [inline HolaMundo.java:6]: The attribute must be named d.

Comment (kamen7375) [inline HolaMundo.java:6]: The attribte must be named d.

**Review by kamen7375 — CHANGES_REQUESTED** [review]
There are some changes that have to be implemented.

File: HolaMundo.java (hunk 1 above)
Comment (Herreran903) [reply HolaMundo.java:6]: Ok, i im in that

**Pull request merged**